desktop/components/main_layout/templates/index.jade

```diff
@@ -1,39 +1 @@
-//- Override any locals with `append locals`
-block locals
-  - bodyClass = helpers ? helpers.buildBodyClass(sd, 'body-header-fixed') : '';
-  - defaultOptions = {modal: true, flash: true, stripe: true, sailthru: true, marketo: true, quantcast: true}
-- options = options ? Object.assign({}, defaultOptions, options) : defaultOptions
-
-doctype html
-html(
-  data-useragent=userAgent
-  data-user-type=sd.CURRENT_USER && sd.CURRENT_USER.type
-  data-lab-features=sd.CURRENT_USER && sd.CURRENT_USER.lab_features && sd.CURRENT_USER.lab_features.join(',')
-  class= sd.HIGHLIGHT_HEADER_SEARCH == 'highlighted' ? 'html-highlight-header-search-split-test' : ''
-)
-  head
-    include head
-    block head
-    link( type='text/css', rel='stylesheet', href=asset('/assets/main_layout.css') )
-    if assetPackage
-      link( type='text/css', rel='stylesheet', href=asset('/assets/#{assetPackage}.css') )
-  body( class=bodyClass )
-
-    //- Global component mixins
-    include ../../modal/template
-    include ../../flash/template
-
-    //- Header, body block, footer
-    include ../header/templates/index
-    #main-layout-container.responsive-layout-container
-      block body
-    #scroll-frame-spinner: .loading-spinner
-    include ../footer/template
-
-    //- Javascripts
-    #scripts
-      include scripts
-      script( src= asset('/assets/main_layout.js') )
-      if assetPackage
-        script( src=asset('/assets/#{assetPackage}.js') )
-      block scripts
+include ./layout/index
```

Comment:
Gotcha. Yeah, it just seems like maybe more indirection than its worth, since it's a one-line file including a three-line file. Could this just serve as the `index` aggregator for the `layout` subdirectory? Maybe moving those partials up to this directory could make sense, if we like to keep the aggregation adjacent to the aggregated files? The fact that they all start with an underscore might differentiate them as well as keeping them in a subdirectory.